doc/source/reference/ufuncs.rst

```diff
@@ -360,6 +360,17 @@ advanced usage and will not typically be used.
     and for generalized ufuncs for which all outputs are scalars, the output
     tuples can be omitted.
 
+*axis*
+
+    .. versionadded:: 1.15
+
+    A single axis over which a generalized ufunc should operate. This is a
```

Comment:
"Not supported for non-generalized ufuncs" might be nice. Does `np.add([1], [2], axis=0)` fail?

Reply:
It does say specifically "over which a *generalized* ufunc should operate. And, yes, an error is raised:
```
In [2]: np.add(1., 1., axis=0)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-578c404b564c> in <module>()
----> 1 np.add(1., 1., axis=0)

TypeError: 'axis' is an invalid keyword to ufunc 'add'
```
But this was not tested - now added.